### PR TITLE
feat: sdk-v47

### PR DESCRIPTION
**Pull request by aljo242**
Fixes QCK-510

Needs a thorough review @joe-bowman @ajansari95 @muku314115

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -7,24 +7,24 @@ COPY go.mod go.sum* ./
 
 RUN --mount=type=cache,target=/root/.cache/go-build \
     --mount=type=cache,target=/root/go/pkg/mod \
-    go mod download
+    go mod download \
```
Comment (joe-bowman) [inline Dockerfile:10]: Incorrect trailing \, causes the RUN to be interpreted by bash.

Comment (joe-bowman) [inline Dockerfile:10]:
```suggestion
    go mod download
```

File: Dockerfile [hunk 2]
```diff
@@ -7,24 +7,24 @@ COPY go.mod go.sum* ./
...
 
 # Cosmwasm - download correct libwasmvm version
 RUN WASMVM_VERSION=$(go list -m github.com/CosmWasm/wasmvm | cut -d ' ' -f 2) && \
     wget https://github.com/CosmWasm/wasmvm/releases/download/${WASMVM_VERSION}/libwasmvm_muslc.$(uname -m).a \
     -O /lib/libwasmvm_muslc.a && \
     wget https://github.com/CosmWasm/wasmvm/releases/download/${WASMVM_VERSION}/checksums.txt -O /tmp/checksums.txt && \
-    sha256sum /lib/libwasmvm_muslc.a | grep $(cat /tmp/checksums.txt | grep $(uname -m) | cut -d ' ' -f 1)
+    sha256sum /lib/libwasmvm_muslc.a | grep $(cat /tmp/checksums.txt | grep libwasmvm_muslc.$(uname -m) | cut -d ' ' -f 1) \
```
Comment (joe-bowman) [inline Dockerfile:17]:
```suggestion
    sha256sum /lib/libwasmvm_muslc.a | grep $(cat /tmp/checksums.txt | grep libwasmvm_muslc.$(uname -m) | cut -d ' ' -f 1)
```

same here.

File: x/airdrop/module.go [hunk 3]
```diff
@@ -115,23 +114,11 @@ func (am AppModule) RegisterInvariants(ir sdk.InvariantRegistry) {
 	keeper.RegisterInvariants(ir, am.keeper)
 }
 
-// Route returns the message routing key for the airdrop module.
-func (am AppModule) Route() sdk.Route {
-	return sdk.Route{}
-}
-
 // QuerierRoute returns the airdrop module's querier route name.
```
Comment (joe-bowman) [inline x/airdrop/module.go:117]: `QuerierRoute` is also removed in 0.47 afaik.

Comment (faddat) [reply x/airdrop/module.go:117]: Yes that's correct

Comment (faddat) [reply x/airdrop/module.go:117]: Sorry I think that could be wrong....

**Review by ajansari95 — APPROVED** [review]
lgtm :shipit:

**Pull request merged**